src/mpid/ch4/include/mpidch4.h

```diff
@@ -232,7 +238,8 @@ MPIDI_CH4I_API(int, Iscatter, const void *, int, MPI_Datatype, void *, int, MPI_
 MPIDI_CH4I_API(int, Iscatterv, const void *, const int *, const int *, MPI_Datatype, void *, int,
                MPI_Datatype, int, MPIR_Comm *, MPIR_Request **);
 
-int MPID_Abort(struct MPIR_Comm *comm, int mpi_errno, int exit_code, const char *error_msg);
+MPIDI_CH4I_API_NOINLINE(int, Abort, struct MPIR_Comm *comm, int mpi_errno, int exit_code,
```

Comment:
Is this macro really necessary? I kind of think less complexity here would be a win for readability...

Reply:
Oh God, not more @charlesarcher macros... 🤢

Reply:
Right. We haven't had to touch these in quite some time. While we're here, just remove them and be done with it once and for all.

Reply:
I added the `MPIDI_CH4I_API_NOINLINE` macro just for consistency.